docs/advanced/experimental/hub.md

```diff
@@ -0,0 +1,63 @@
+# Hub Portal
+
+Hub is a marketplace for Executors. It allows everyone to share their Executors and use others' Executors. You can check the previous [tutorial](/advanced/hub/) if you want to rehearse the creation of an Executor.
```

Comment:
It is recommended to use the new `{ref}` syntax instead of relying on paths. Check its usage in other pages. It's a safer way to maintain links

Reply:
Where is the `{ref}` syntax from? I searched it but seems neither in pradyunsg.me nor markdown. Just wonder how it works.

Reply:
https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#targets-and-cross-referencing